dotnet/src/Microsoft.Agents.AI.Hosting.OpenAI/Responses/Generated/Models/ComputerActionScreenshot.cs

```diff
@@ -0,0 +1,26 @@
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Agents.AI.Hosting.OpenAI.Responses.Generated.Models
+{
+    /// <summary> A screenshot action. </summary>
+    internal partial class ComputerActionScreenshot : ComputerAction
```

Comment:
There are a ton of types being added, presumably all autogenerated from the typespec, but many of which we likely will never use, like this one. Do we need to add them all? Would it be possible to have them consolidated into fewer files, rather than adding literally 600 files?

Reply:
Could we group them into a single file at least?

Reply:
I've updated the PR to replace the generated models